config/routes.rb

```diff
@@ -2,4 +2,48 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  #MVP - Access to the dashboad
+  get 'dashboard', to: 'dashboard#show'
+
+  # MVP - Create a garden and plots into the garden
+  resources 'gardens', only: [:new, :create] do
+    resources 'plots', only: [:new, :create]
+  end
+
+  # MVP- Update and destroy plots
+  resources 'plots', only: [:update, :edit, :destroy] do
+    #MVP - create plants inside a plot
+    resources 'plants', only: [:create]
+  end
+  #MVP - Watering see the all the watering, one specific watering and update
+  resources 'waterings', only: [:index, :show, :update]
+
```

Comment:
make custom path: patch waterings/:id/complete, to: waterings#mark_as_complete

Reply:
done